exercises/practice/ledger/src/main/java/Ledger.java

```diff
@@ -106,12 +106,23 @@ public String format(String cur, String loc, LedgerEntry[] entries) {
                     count++;
                 }
 
-                amount = curSymb + amount + decSep + parts[1];
-
-                if (e.getChange() < 0) {
-                    amount = "-" + amount;
+                if (loc.equals("nl-NL")) {
+                    amount = curSymb + " " + amount + decSep + parts[1];
+                } else {
+                    amount = curSymb + amount + decSep + parts[1];
                 }
+                
 
+                if (e.getChange() < 0 && loc.equals("en-US")) {
+                    amount = "(" + amount + ")";
+                } else if (e.getChange() < 0 && loc.equals("nl-NL")) {
+                    amount = curSymb + " -" + amount.replace(curSymb, "").trim() + " ";
+                } else if (loc.equals("nl-NL")) {
+                    amount = " " + amount + " ";
+                } else {
+                    amount = amount + " ";
+                }
+                
```

Comment:
Due to the large diff in `LedgerTest.java` I'm not 100% sure what has changed in the behavior of this exercise, can you explain why these changes were needed?

Reply:
Yes! well first of all this is a reformat exercise, so I have to match the main code that receives the user with the reference to pass the new tests.
The new synced tests changed:
- They pass an ``int`` instead of a ``double``
- They asked for a different format of negative values in ``en-US`` ``($10.00)``
- Also if the number is positive to always have a space after the number
- And for ``nl-NL`` a space between the currency and the number